### DISCUSSION TITLE
Can I embed other gui framework in korge?

### DISCUSSION THREAD

**Discussion by Anivie** [korlibs/korge#939] [Q&A]
Like JavaFX?So i can use a easy way to build a ui for my game.

**Comment by soywiz** [comment]
I'm afraid you cannot embed JavaFX. You can use AWT and embed korge as an awt component though. You have an example of this here: https://github.com/korlibs/korge/blob/f448b148c5d1da60e32359df03538f1ecf118473/korge-sandbox/src/jvmMain/kotlin/AwtSandboxSample.kt#L45

In any case, my suggestion is to use the UI components KorGE provides: https://github.com/korlibs/korge/tree/f448b148c5d1da60e32359df03538f1ecf118473/korge/src/commonMain/kotlin/com/soywiz/korge/ui

There is also a pending PR for integrating JetPack compose inside KorGE: https://github.com/korlibs/korge/pull/485 not merged yet until it is stable enough.
